parm/atm/berror/staticb_gsibec.yaml

```diff
@@ -0,0 +1,39 @@
+background error:
+  covariance model: hybrid
+  components:
+  - covariance:
+      covariance model: SABER
+      saber blocks:
+      - saber block name: gsi covariance
+        saber central block: true
+        input variables: &control_vars [eastward_wind,northward_wind,air_temperature,surface_pressure,
+                                 specific_humidity,cloud_liquid_ice,cloud_liquid_water,
+                                 mole_fraction_of_ozone_in_air]
+        output variables: *control_vars
+        gsi akbk: &akbk $(fv3jedi_fix_dir)/akbk.nc4
+        gsi error covariance file: &gsiberr $(fv3jedi_staticb_dir)/gsi-coeffs-gfs-global-l127x192y96.nc4
+        gsi berror namelist file: &gsibnml $(fv3jedi_staticb_dir)/gfs_gsi_global.nml
+        processor layout x direction: 3
```

Comment:
Question: is this like FV3 where this has to be consistent with the MPI task count? Are these additional PEs to the FV3-JEDI ones? Or does it not matter?

Reply:
I have the same question.  Testing indicates that the analysis increments change when we change the [3,2] layout.   I tried [2,3] and [1,6].   `fv3jedi_var.x` ran to completion.  `compare_ncfile.py` of the increment files found non-zero differences.  I also tried [1,1].   `fv3jedi_var.x` failed.  This is not unexpected.   It looks like the product of the layout must equal the number of tasks with which `fv3jedi_var.x` is run.   The layout is an item on list of gsibec questions for Ricardo.

Reply:
Hmm, okay, so then these are going to need to be variables too then, as they will need to be consistent with `layout_x` and `layout_y` (but not the same as...)

Reply:
I agree.  I do not know how to answer the "_consistent but not the same as_" question.   Let me reach out to Ricardo.

Reply:
Thank you, @KateFriedman-NOAA and @CoryMartin-NOAA for noting C766 -vs- C768.  Let me look at this and get back to you.

Reply:
You are right @CoryMartin-NOAA and @KateFriedman-NOAA .   Directory `C766` should be `C768`.   I moved `C766` to `C768` in Hera `/scratch1/NCEPDEV/da/Russ.Treadon/GDASApp/fix/gsibec`.

Reply:
"Differences in ua and va increments are order 5 m/s. T differences are around 3 K." Yikes.

Reply:
Yes, quite disturbing!